### ISSUE TITLE
Issue with two tags

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/TaxoPress#2056]
In the Auto Links feature, there's a problem if you have two tags with the same word.

For example: "London" and "London News".

Auto Links defaults to choosing the shortest tag ("London") and ignores the longer tag ("London News") even if is repeated later in the text

**Comment by ojopaul** [comment]
@stevejburge Auto Link will match the first term. So, if there's "London" Term and there is "London News", Isn't it logical that the first term will be tagged or how do you see this working?

**Comment by stevejburge** [comment]
@ojopaul Yes, I think "London" should be processed first. The alternative would be create a priority system.

The customer problem was that "London News" was then ignored later in the post.

**Comment by ojopaul** [comment]
On the priority, you can decide that by reordering the terms. There's no gimmicks to that, whatever come first will be tagged first but if you want something to have high priority, reorder the tags and make them come first so they'll always be first in the result.

Now on the "London News" been ignore, do you mean there's a "London News" where it London wasn't tagged yet the whole "London News" still fail to be tagged?

If so, I believe this has to do with the configuration like limit etc, do you mind getting me a site to confirm the problem isn't with the configuration as under normal circumstances, if there's a whole "London News" that neither of the part is previously tagged, there's no reason for it not to be tagged when it reach the London News tag? @stevejburge
